src/taco-cli/cli/utils/settings.ts

```diff
@@ -87,7 +87,7 @@ class Settings {
             Settings.determinePlatformsFromOptions(options)
         ]).spread<Settings.IPlatformWithLocation[]>(function (supportedPlatforms: CordovaHelper.IDictionary<any>, platforms: Settings.IPlatformWithLocation[]): Settings.IPlatformWithLocation[] {
             var filteredPlatforms = platforms.filter(function (platform: Settings.IPlatformWithLocation): boolean {
-                var supported = !supportedPlatforms || platform.platform in supportedPlatforms;
+                var supported = !supportedPlatforms || platform.platform in supportedPlatforms || platform.location === Settings.BuildLocationType.Remote;
```

Comment:
Correction: This is about filtering out non-platforms, not platforms that aren't supported on this host.

The user could specify "taco build sndroid" or "taco build foo" and we know for a fact that cordova will not build these non-platforms, so we are filtering them out here. It also protects against non-platform folders in /platforms